### PR TITLE
Adhikram group data

### PR DESCRIPTION
**Pull request by Adhikram**

### CODE REVIEW

File: devsnest/app/models/group.rb [hunk 1]
```diff
@@ -144,4 +144,40 @@ def invite_inactive_members
...
+      scrum_attended = scrums_data.where(attended: true).count
+
+      total_assignments_challenge_ids = AssignmentQuestion.where(course_curriculum_id: course_curriculum_ids, question_type: 'Challenge').pluck(:question_id)
+      solved_assignments_count = UserChallengeScore.where(user_id: user_id, challenge_id: total_assignments_challenge_ids).count
+
+      recent_total_assignments_ch_ids = AssignmentQuestion.where(course_curriculum_id: course_curriculum_ids, question_type: 'Challenge')
+                                                          .where('created_at > ?', (Date.today - 15.days).beginning_of_day).pluck(:question_id)
+      recent_solved_assignments_count = UserChallengeScore.where(user_id: user_id, challenge_id: recent_total_assignments_ch_ids).count
+
+      result << {
```
Comment (vedansh) [inline devsnest/app/models/group.rb:168]: looks like old way to add data to an array

Comment (Adhikram) [reply devsnest/app/models/group.rb:168]: Okay … but it's working

Comment (LakshitXD) [reply devsnest/app/models/group.rb:168]: apply caching on this.

Comment (Adhikram) [reply devsnest/app/models/group.rb:168]: okay

**Review by vedansh — APPROVED** [review]

**Pull request merged**